### ISSUE TITLE
getServerString() with no port

### ISSUE THREAD

**Issue by Jpgemy** [ddeboer/imap#311]
Hi,
I need to let the port blank sometimes.

So can you change this : 
```php
private function getServerString(): string
    {
        return \sprintf(
            '{%s:%s%s}',
            $this->hostname,
            $this->port,
            $this->flags
        );
    }
```

with this : 

```php
private function getServerString(): string
    {
        return \sprintf(
            '{%s%s%s}',
            $this->hostname,
            empty($this->port) ? '' : ':' . $this->port,
            $this->flags
        );
    }
```

Thanks

**Comment by Slamdunk** [comment]
You can pass an empty string to the constructor.

**Comment by Slamdunk** [comment]
Oh sorry, I just saw the colon, got it :+1:

**Comment by Jpgemy** [comment]
So the server string is "imap.server.tld:/notls".

I don't want the ":" because it doesn't work with. And I can't remove it with str_replace or something else because all the connection is done in the "authenticate()" method.

**Comment by Slamdunk** [comment]
Yup, got it :)

**Comment by Jpgemy** [comment]
Yes, thanks ! ;)

**Comment by Slamdunk** [comment]
Released in [1.3.1](https://github.com/ddeboer/imap/releases/tag/1.3.1)
